src/defaultsettings.cpp

```diff
@@ -57,6 +57,10 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("curl_verify_cert", "true");
 	settings->setDefault("enable_remote_media_server", "true");
 	settings->setDefault("enable_client_modding", "false");
+	settings->setDefault("enable_knockback", "true");
+	settings->setDefault("knockback_default_duration", "0.3");
```

Comment:
exact, it's a predicted knockback, which doesn't care about lag, we don't force the knockbacked client to send the movements to server to redispatch it to other clients. Server send a knockback order and all clients do the animation, and the affected player has his controls blocked during the animation + do no sent MOVE packets during this moment